### PR TITLE
Added checksums for newer versions of numpy

### PR DESCRIPTION
**Pull request by s-sajid-ali**
Added checksums for newer versions of numpy! 

	modified:   var/spack/repos/builtin/packages/py-numpy/package.py

### CODE REVIEW

**Comment by mamelara** [comment]
flake8: var/spack/repos/builtin/packages/py-numpy/package.py:50: [W293] blank line contains whitespace

File: var/spack/repos/builtin/packages/py-numpy/package.py [hunk 1]
```diff
@@ -47,7 +47,10 @@ class PyNumpy(PythonPackage):
 
     # FIXME: numpy._build_utils and numpy.core.code_generators failed to import
     # FIXME: Is this expected?
-
+    
+    version('1.14.2', '080f01a19707cf467393e426382c7619')
+    version('1.14,1', 'b8324ef90ac9064cd0eac46b8b388674')
```
Comment (healther) [inline var/spack/repos/builtin/packages/py-numpy/package.py]: this should be `'1.14.1'` right?

**Comment by adamjstewart** [comment]
Unfortunately I can't merge the PR until the flake8 tests pass. See http://spack.readthedocs.io/en/latest/contribution_guide.html#flake8-tests for more details.

File: var/spack/repos/builtin/packages/py-numpy/package.py [hunk 2]
```diff
@@ -47,7 +47,10 @@ class PyNumpy(PythonPackage):
 
     # FIXME: numpy._build_utils and numpy.core.code_generators failed to import
     # FIXME: Is this expected?
-
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-numpy/package.py]: This line still has spaces at the beginning. You should be able to configure your text editor to automatically remove trailing whitespace.

**Pull request merged**